Sources/SwiftStylable/Classes/Style/StylableComponents/STStackView.swift

```diff
@@ -7,38 +7,24 @@
 
 import UIKit
 
-@IBDesignable open class STStackView: UIStackView {
-
-    private var _dimension:String?
+@IBDesignable open class STStackView: UIStackView, Stylable, StackViewSpacingStylable {
```

Comment:
Ik denk dat StackViewSpacingStylable wat te specifiek is. De '...Stylable' typen zijn overal zo generiek mogelijk opgezet, met bijvoorbeeld wel TextStylable, maar iet specifiek labelTextStylable, zodat hij inzetbaar is voor (bijna) alles met text. Er zijn uitzonderingen, zoals wat gedoe met de TableViewSeparator, omdat die ook specifiek refereert naar onderliggende enums die zo heten (en eigenlijk zou je je bij tableViewSeparatorColor) af kunnen vragen of het niet beter separatorColor had kunnen heten.). Aangezien de waarde hier gewoon een CGFloat (dimension) is, denk ik dat hij beter gewoon 'SpacingStylable' kan heten. (Met ook de property 'spacing')